girder/utility/webroot.py

```diff
@@ -90,17 +90,32 @@ def __init__(self, templatePath=None):
             'plugins': [],
             'apiRoot': '',
             'staticRoot': '',
+            # 'title' is depreciated use brandName instead
             'title': 'Girder',
+            'brandName': ModelImporter.model('setting').get(SettingKey.BRAND_NAME),
             'contactEmail': ModelImporter.model('setting').get(SettingKey.CONTACT_EMAIL_ADDRESS)
         }
 
         events.bind('model.setting.save.after', CoreEventHandler.WEBROOT_SETTING_CHANGE,
                     self._onSettingSave)
+        events.bind('model.setting.remove', CoreEventHandler.WEBROOT_SETTING_CHANGE,
+                    self._onSettingRemove)
 
     def _onSettingSave(self, event):
         settingDoc = event.info
         if settingDoc['key'] == SettingKey.CONTACT_EMAIL_ADDRESS:
             self.updateHtmlVars({'contactEmail': settingDoc['value']})
+        elif settingDoc['key'] == SettingKey.BRAND_NAME:
+            self.updateHtmlVars({'brandName': settingDoc['value']})
+
+    def _onSettingRemove(self, event):
+        settingDoc = event.info
+        if settingDoc['key'] == SettingKey.CONTACT_EMAIL_ADDRESS:
+            self.updateHtmlVars({'contactEmail': ModelImporter.model(
+                'setting').getDefault(SettingKey.CONTACT_EMAIL_ADDRESS)})
+        elif settingDoc['key'] == SettingKey.BRAND_NAME:
+            self.updateHtmlVars({'brandName': ModelImporter.model(
```

Comment:
Putting the line break here is a little awkard for me to read. How about:
```
self.updateHtmlVars({
    'contactEmail': ModelImporter.model('setting').getDefault(
        SettingKey.CONTACT_EMAIL_ADDRESS)})
```
That is admittedly 1 line longer, but fits inside 100 columns and is easier to read (for me).

Reply:
I'm ok with this change